### PR TITLE
Holtan code review2

### PR DESCRIPTION
**Pull request by adamholt**
Overall the project so far I am quite impressed with! After getting docker installed and working, the project looks and operates smoothly, all classes/interfaces/files etc cleanly formatted and sorted,  and was difficult to find changes that needed to be made. 
When it came to the tests page, there was a second copy of them that I removed along with the qualifier annotation needing the TaskServiceStub, as that class no longer existed. I chose not to do much else with the tests page as I already covered those in my first review, and they were not merged. 
The another change I made was adding a Cancel button to the taskform after clicking addTask, in case a user needed to cancel an add without having to add then delete the task (does not fully function, apologies in advance!). 
The final change made was to forego the navbar. I know in the first sprint review I suggested it, however after seeing the layout and operation of the project, I found it unneeded (unless managing a large number of tasks) as everything was clearly displayed/readable to the user without a search.

After reviewing your project I learned:
more about and solidified concepts about thymeleaf and fragments, there were many applications of them in this project!
about Redirect URLs
about Docker Hub

### CODE REVIEW

File: src/main/resources/templates/taskform.html [hunk 1]
```diff
@@ -58,6 +58,11 @@
                     <input type="submit" value="Save"/>
                 </div>
             </div>
+            <div class="task-field">
+                <div class="button">
+                    <input type="button" value="Cancel"/>
```
Comment (discospiff) [inline src/main/resources/templates/taskform.html]: Don't add this, until it is fully functional.  You shouldn't merge incomplete features into master.

**Comment by discospiff** [comment]
Many of the changes here are features, either partial additions or removals, as opposed to best practice recommendations for code.  I'm reluctant to recommend merging, and will leave it up to the group to make the final call.  See inline comments for more details.

File: src/main/resources/templates/taskform.html (hunk 1 above)
Comment (joeboylson) [reply src/main/resources/templates/taskform.html]: Thanks for the tip. 
Its a good thought, I'll add to this feature and then go ahead and merge it in.

**Comment by joeboylson** [comment]
@adamholt Thanks for your PR.
A few of your changes I will probably revert (such as removing the search modal) but I will go ahead and complete and merge in your cancel button suggestion.
Thanks for your thoughts and feedback on this project.

**Pull request merged**